notte/common/credentials/prompt.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
+
+
+def get_credentials_prompt() -> str:
```

Comment:
`prompt.py` and `system.md` are too short to justify sperate files. Can you simply add a `def instructions(self) -> str`  or `def system` in `BaseVault` directely ?